README.md

```diff
@@ -1 +1,2 @@
-# GitPractice
\ No newline at end of file
+# GitPractice
+My first project.
```

Comment:
Ar fi trebuit sa fie scris in limba romana.

Reply:
mews

Reply:
Am rezolvat. Merci :)